src/main/java/uk/gov/hmcts/reform/laubackend/cases/utils/InputParamsVerifier.java

```diff
@@ -94,11 +94,10 @@ public static void verifyRequestSearchCaseRefs(final List<String> caseRefs) {
 
     public static void verifyAccessRequestGetTimestamp(final String timestamp) throws InvalidRequestException {
         try {
-            if (timestamp == null) {
-                throw new InvalidRequestException(TIMESTAMP_GET_EXCEPTION_MESSAGE, HttpStatus.BAD_REQUEST);
+            if (timestamp != null) {
```

Comment:
I would suggest to do this check in timestamp utils class. In addition, app crashes once you try to retrieve null timestamp from DB.

Reply:
This has been addressed